### DISCUSSION TITLE
Force Full Show when requesting TV series

### DISCUSSION THREAD

**Discussion by Foebik** [sct/overseerr#1250] [Q&A] [answered]
Is there a way to take the option away from the user that lets them just pick and choose which season they are requesting and will instead request all the seasons in a show?

I hate having partial tv series and if a user request something I want all the seasons to be downloaded by default.

**Comment by sct** [comment, marked as answer]
This feature is already available in the develop branch and will be in the next release.

**Comment by Foebik** [comment]
Thanks again @sct
